src/main/java/com/puppycrawl/tools/checkstyle/checks/OuterTypeFilenameCheck.java

```diff
@@ -38,7 +38,18 @@
  * <pre>
  * &lt;module name=&quot;OuterTypeFilename&quot;/&gt;
  * </pre>
+ *<p>Example of class Test in a file named Test.java</p>
+ * <pre>
+ * public class Test { // OK
  *
+ * }
+ * </pre>
+ * <p>Example of class Foo in a file named Test.java</p>
```

Comment:
Please also add examples for interface and enum as well

Reply:
done

Reply:
@HuGanghui , please combine all that examples in one PRE block.